src_py/__init__.py

```diff
@@ -28,8 +28,14 @@
 
 # Choose Windows display driver
 if os.name == "nt":
+    pygame_dir = os.path.split(__file__)[0]
+
     # pypy does not find the dlls, so we add package folder to PATH.
-    os.environ["PATH"] = os.environ["PATH"] + ";" + os.path.split(__file__)[0]
+    os.environ["PATH"] = os.environ["PATH"] + ";" + pygame_dir
+
+    # windows store python does not find the dlls, so we run this
+    if sys.version_info > (3, 8):
```

Comment:
this should be `>=`

Reply:
It will never be equal. version_info is a 5 element named tuple.

```py
>>> a = (3, 8, 0, 'final', 0)
>>> a > (3, 8)
True
```

Reply:
Oh, yeah that makes sense. For some reason I thought that `version_info` would be a custom tuple subclass that would handle comparisons differently, but that's not the case